pkgs/applications/science/chemistry/mopac/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, gfortran
+, fetchFromGitHub
+, cmake
+, blas
+, lapack
+, python3Packages
+}:
+
+assert blas.isILP64 == lapack.isILP64;
```

Comment:
Does the package automatically detect ILP64 (I do not see that here)? Maybe we should also assert `isILP64 == false`.

Reply:
It actually seems to do, yes. We already figured this out in NixOS-QChem: https://github.com/markuskowa/NixOS-QChem/pull/217#discussion_r989067288

Reply:
Oh, ok, never mind then.